go-selinux/selinux_linux.go

```diff
@@ -781,6 +781,20 @@ func selinuxEnforcePath() string {
 	return filepath.Join(getSelinuxMountPoint(), "enforce")
 }
 
+func selinuxMLSEnabledPath() string {
+	return filepath.Join(getSelinuxMountPoint(), "mls")
+}
+
+// isMLSEnabled checks if MLS is enabled.
+func isMLSEnabled() bool {
+	enabledB, err := os.ReadFile(selinuxMLSEnabledPath())
+	if err != nil {
+		return false
+	}
+
+	return !bytes.Equal(enabledB, []byte{'1'})
```

Comment:
Should this be the reverse? (i.e., it return `true` if the file contains `1` ?

Reply:
It is weird but apparently not https://github.com/SELinuxProject/selinux/blob/3f11c7d19c449996d7c34190d6aa74220b104779/libselinux/src/enabled.c#L51

Reply:
Ooooh!

```
/*
 * Function: is_selinux_mls_enabled()
 * Return:   1 on success
 *	     0 on failure
 */
```

Hm.. right; perhaps would be good to add a comment there to describe it (perhaps also with that link) to explain this is really correct 😅.

Reply:
@giuseppe yes you're right

Reply:
i was going through this line and documentation of strcmo multiple times I have no idea how I convoced myself to be wrong so many times

Reply:
OH MY! 🤦 for some reason I completely didn't spot that as well. Thanks @giuseppe !!! (it adds to my dislike of "too many booleans")

Reply:
fixed